server/channels/app/post.go

```diff
@@ -1901,6 +1901,11 @@ func isCommentMention(user *model.User, post *model.Post, otherPosts map[string]
 		return mentioned
 	}
 
+	if _, ok := otherPosts[post.RootId]; !ok {
+		// RootPost is past the cloud plan's limit.
```

Comment:
I'm not sure if we should log a warning/error into the logs if this happens. The benefits I see from that is:
- people will know why something is not loading
- if they check the logs they get a new incentive into a paid plan